app/helpers/application_helper.rb

```diff
@@ -573,6 +570,59 @@ def permitted_params
 
   private
 
+  def render_hierarchy_item(page, is_parent, options = {})
+    content_tag(:span, class: 'tree-menu--item', slug: page.slug) do
+      hierarchy_span_content = nil
+      if is_parent
+        hierarchy_span_content = if is_parent
+                                   render_hierarchy_indicator_icons
+                                 else
+                                   render_leaf_indicator
+                                 end
+      end
+      concat content_tag(:span, hierarchy_span_content, class: 'tree-menu--hierarchy-span')
+
+      concat link_to(page.title,
+                     project_wiki_path(page.project, page),
+                     title: hierarchy_item_title(options, page),
+                     class: 'tree-menu--title ellipsis') do
+      end
+    end
+  end
+
+  def hierarchy_item_title(options, page)
+    if options[:timestamp] && page.updated_on
+      l(:label_updated_time, distance_of_time_in_words(Time.now, page.updated_on))
```

Comment:
Prefer standard I18n's `t(:...)` over redmine's magic `l(:....)`.

Reply:
@oliverguenther Yeah, but I18n's @t(:...)@ didn't work here. Might be as this is a date/time localisation and thus a different story than translation?

Reply:
It will work if you use `t(:label_updated_time, value: distance_of_time...)`.